Runtime/Scripts/EdgeEventsConfig.cs

```diff
@@ -1,123 +1,157 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
 using UnityEngine;
 using DistributedMatchEngine;
 using System.Collections.Generic;
 
 namespace MobiledgeX
 {
-    [Serializable]
-    public struct EdgeEventsConfig
-    {
-        /// <summary>
-        /// Latency threshold (In Milliseconds) as the limit to automatically migrate to a new cloudlet
-        /// </summary>
-        [Tooltip("Latency threshold (In Milliseconds) as the limit to automatically migrate to a new cloudlet")]
-        public double latencyThresholdTriggerMs;
-        /// <summary>
-        /// Port information for latency testing
-        /// <para>Use 0 to select a random port</para>
-        /// </summary>
-        [Tooltip("Port information for latency testing" +
-            "\nUse 0 to select a random port")]
-        public int latencyTestPort;
-        /// <summary>
-        /// Config for latency updates
-        /// </summary>
-        [Tooltip("Config for latency updates")]
-        public UpdateConfig latencyConfig;
-        /// <summary>
-        /// Config for location updates
-        /// </summary>
-        [Tooltip("Config for location updates")]
-        public UpdateConfig locationConfig;
-
-        /// <summary>
-        /// List of triggers that will trigger a new find cloudlet.
-        /// </summary>
-        [Tooltip("List of triggers that will trigger a new find cloudlet.")]
-        public List<FindCloudletEventTrigger> newFindCloudletEventTriggers;
-    }
+  [Serializable]
+  public class EdgeEventsConfig
+  {
+    /// <summary>
+    /// Latency threshold (In Milliseconds) as the limit to automatically migrate to a new cloudlet
+    /// </summary>
+    [Tooltip("Latency threshold (In Milliseconds) as the limit to automatically migrate to a new cloudlet")]
+    public double latencyThresholdTriggerMs;
+    /// <summary>
+    /// Port information for latency testing
+    /// <para>Use 0 to select a random port</para>
+    /// </summary>
+    [Tooltip("Port information for latency testing" +
+        "\nUse 0 to select a random port")]
+    public int latencyTestPort;
+    /// <summary>
+    /// Config for latency updates
+    /// </summary>
+    [Tooltip("Config for latency updates")]
+    public UpdateConfig latencyConfig;
+    /// <summary>
+    /// Config for location updates
+    /// </summary>
+    [Tooltip("Config for location updates")]
+    public UpdateConfig locationConfig;
+    /// <summary>
+    /// List of triggers that application that will trigger a new find cloudlet.
+    /// </summary>
+    [Tooltip("List of triggers that application that will trigger a new find cloudlet.")]
```

Comment:
Kind of awkward. Suggestion:
List of triggers for the application to perform a new find cloudlet.